### DISCUSSION TITLE
如何同步到最新版本？

### DISCUSSION THREAD

**Discussion by lyizhuo** [spencerwooo/onedrive-vercel-index#390] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/64514694/153180810-7c3747a7-01b5-4416-b55e-ff9b9b3c55bc.png)

**Comment by People-11** [comment, marked as answer]
请参考[文档](https://onedrive-vercel-index.spencerwoo.com/zh/docs/migration/updating-to-latest-version) 
我通常使用 Sublime Merge 软件解决冲突。

**Comment by spencerwooo** [comment]
如果 conflict 太多建议重新 fork 一版本，有时候从头再来更方便。
